Add test: ERR002.001 PC speaker beep during boot (firmware) (#758)
Dasharo/docs: docs/unified-test-documentation/dasharo-compatibility/31S-sata-led-and-pc-speaker-error-indication.md

be triggered causing the platform to beep and blink the SATA LED. There should
be only 12 beeps, but the SATA LED should blink endlessly (until DUT is powered
off or reset).

## ERR002.001 PC speaker beep during boot (firmware)

**Test description**

> This test aims to verify if PC speaker beep during the boot process.

Reviewer: ```suggestion
This test aims to verify if PC speaker beeps during the boot process.
```

Author: Done: https://github.com/Dasharo/docs/pull/758/files#diff-17612c7451237a26967643a8c1e896e1f5397447ac3236ad51b53103fb0003f2R41

**Test configuration data**

1. `FIRMWARE` = Dasharo
2. DUT with PC speaker

**Test setup**

1. Proceed with the
    [Generic test setup: firmware](../generic-test-setup.md#firmware).

**Test steps**

1. Power on the DUT.

> 2. Listen for a single beep when the bootsplash logo appears.

Reviewer: ```suggestion
2. Listen for a single beep until the bootsplash logo appears.
```

**Expected result**

> The DUT should single beep when the bootsplash logo appears.

Reviewer: ```suggestion
The DUT should make a single beep shortly before the bootsplash logo appears.
```

Author: Done: https://github.com/Dasharo/docs/pull/758/files#diff-17612c7451237a26967643a8c1e896e1f5397447ac3236ad51b53103fb0003f2R60
